splash.py

```diff
@@ -6,6 +6,11 @@
 def splash(i, o):
     image = PIL.Image.open("splash.png").convert('L')
     image = invert(image)
-    image = image.convert(o.device.mode)
-    o.device.display(image)
+    #FIXME:
+    # need a first-class API call on the display to do this
```

Comment:
Agreed. I started work on this [here](https://github.com/ZeroPhone/pyLCI/commit/74e2dbe8aa369268257913641c558b67f38e8a3e) some time ago - as part of work on #3, in prettier_ui branch.